### ISSUE TITLE
App can't start

### ISSUE THREAD

**Issue by chuan0418-com** [aredden/electron-G14Control#80]
![image](https://user-images.githubusercontent.com/52336350/148681982-9ba50616-f724-4dfb-b302-907c748b961a.png)
I can't start the app, it pops up this error.
I'm using windows 11, and it actually work fine before days.

**Comment by aredden** [comment]
Did you start with admin permissions?

**Comment by chuan0418-com** [comment]
> Did you start with admin permissions?

Yes. I'd even reinstall serveral times and it didn't work for me.

**Comment by aredden** [comment]
I'm thinking it might have to be something like.. the "Allow local powershell scripts to run without signing" windows setting?

**Comment by chuan0418-com** [comment]
`        Scope ExecutionPolicy`
`        ----- ---------------`
`MachinePolicy       Undefined `
`   UserPolicy       Undefined`
`      Process       Bypass`
`  CurrentUser          Bypass`
` LocalMachine          Bypass`


I enter `Get-ExecutionPolicy -List` and it show up these setting.
But the app still show up the error. Is my setting correct?

**Comment by aredden** [comment]
Well I know it's some sort of permission issue, since that's what an EPERM error is. It's probably related to powershell. Do you not have powershell installed? This is the file in question: https://github.com/aredden/electron-G14Control/blob/main/electron/src/IPCEvents/ComputerInfo.ts ..

**Comment by aminbahjati** [comment]
<img width="418" alt="image" src="https://user-images.githubusercontent.com/26881073/151978000-2a71446b-f22c-4847-9f7b-3d434b2191be.png">
<img width="419" alt="image" src="https://user-images.githubusercontent.com/26881073/151978786-425ce1a3-becd-4ad0-a288-203f5853eaf6.png">

I faced this problem today and these two errors raised.
What should I do?
